### ISSUE TITLE
available?

### ISSUE THREAD

**Issue by redSonja113** [dukeofharen/shortnr.net#1]
Hi, I'm learning to create url shorteners right now and came upon your app. Is the tutorial still available anywhere?

**Comment by dukeofharen** [comment]
Hi,

Thanks for your interest in this tutorial. It's taken offline a little while ago indeed. I looked in my archive and found the original tutorial. I've saved it as PDF and can be downloaded here: https://file.io/OCNIL8

I have to say, however, that I created this tutorial when I didn't nearly have as much experience as a developer yet. Also, this tutorial is meant for developing a URL shortener using ASP.NET, and not ASP.NET Core (which is the framework you'll likely use if you're getting started with ASP.NET nowadays).
Aside from that, I hope you like the article and can still use it to learn something new.

Kind regards,
Duke
